resources/js/Pages/Login.vue

```diff
@@ -3,6 +3,12 @@
     <Head title="Login" />
     <Menu />
     <section>
+        <b-message
+                    type="is-danger"
+                    v-if="$page.props.flash.error">
+                        {{ $page.props.flash.error }}
+
+        </b-message>
         <div class="box">
             <form action="/loginCheck" method="POST" @submit.prevent="login">
```

Comment:
```suggestion
            <form @submit.prevent="login">
```
Unneeded attributes because we use Inertia